src/main/java/com/Timos/TicTacToeInputProducer/Service/TicTacToeInputService.java

```diff
@@ -0,0 +1,16 @@
+package com.Timos.TicTacToeInputProducer.Service;
+
+import com.Timos.TicTacToeInputProducer.model.InputMessage;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+
+@Service
+public class TicTacToeInputService {
```

Comment:
Why do I need this service on top of the `TicTacToeInputKafkaProducerService`? It doesn't seem to add any logic